### DISCUSSION TITLE
Support "IntelliSense" plugin for JetBrains WebStorm

### DISCUSSION THREAD

**Discussion by thanhtoan1196** [tailwindlabs/tailwindcss#2851] [Ideas]
Please support "IntelliSense" plugin for JetBrains WebStorm.

**Comment by MartinHughes-BPC** [comment]
As I understand it, JetBrains are releasing their own plugin for Tailwind support. Checkout this link: https://blog.jetbrains.com/webstorm/2020/11/webstorm-2020-3-eap-7/

Seems it only works on EAP releases of WebStorm (etc.) at the moment.

**Reply by ihorvorotnov** [reply]
2020.3 is already out and this new bundled plugin is awesome 👍
